pkg/clients/status.go

```diff
@@ -19,6 +19,8 @@ const (
 	EnabledStatus Status = iota
 	// DisabledStatus represents disabled Client.
 	DisabledStatus
+	// DeletedStatus represents a client that will be deleted.
+	DeletedStatus
```

Comment:
`DeleteStatus` can be highest status , If we want to hide the Deleted users from List with status=all `http://localhost/users?status=all`, 
If DeleteStatus becmes higher status, then superadmin can get deleted user by explicitly like `status=deleted`

Reply:
Highest status meaning 0 or n?

Reply:
`n`